src/ray/raylet/node_manager.cc

```diff
@@ -1554,13 +1559,28 @@ void NodeManager::HandleWorkerAvailable(const std::shared_ptr<WorkerInterface> &
   cluster_task_manager_->ScheduleAndDispatchTasks();
 }
 
+void SendDisconnectClientReply(const WorkerID &worker_id,
+                               const std::shared_ptr<ClientConnection> &client) {
```

Comment:
`ClientConnection&`

Reply:
in order to maintain consistency I'll prefer to leave it as-is for now, but we should follow up to revisit the usage of const refs for shared pointers in this file. I think it's also widely used in core worker